plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -437,6 +483,21 @@ runThreeArgumentModel
     (ModelThreeArgumentsLinearInZ (OneVariableLinearFunction intercept slope)) =
         lazy $ \_ _ costs3 ->
             scaleLinearly intercept slope costs3
+runThreeArgumentModel
+    (ModelThreeArgumentsQuadraticInZ f) =
+        lazy $ \_ _ costs3 -> CostLast $ evaluateOneVariableQuadraticFunction f $ sumCostStream costs3
+{- Either a literal number of bytes or a linear function.  This is for
+   `integerToByteString`, where if the second argument is zero, the output
+   bytestring has the minimum length required to contain the converted integer,
+   but if the second argument is nonzero it specifies the exact length of the
+   output bytestring. -}
+runThreeArgumentModel
+    (ModelThreeArgumentsLiteralInYOrLinearInZ (OneVariableLinearFunction intercept slope)) =
+        lazy $ \_ costs2 costs3 ->
+            let width = sumCostStream costs2
+            in if width == 0
```

Comment:
Measuring `width` in costing units is extremely weird in my opinion and further reinforces the fact that our current approach needs revising.

Reply:
> Measuring width in costing units is extremely weird in my opinion

Yes, I hate this.  Sizes in bytes would be much better (but it's tricky to change that at the moment).